### PR TITLE
feat(sidebar): add sidebar order tracking

### PR DESCRIPTION
**Pull request by neolitec**
SCFRCA-1045

**Don't merge until _ovh-angular-actions-menu_ and _ovh-angular-sidebar-menu_ are merged!**
See https://github.com/ovh-ux/ovh-angular-actions-menu/pull/10 and https://github.com/ovh-ux/ovh-angular-sidebar-menu/pull/23

Use the new click handler on sidebar menu library to track click on order menu.

### CODE REVIEW

**Comment by antleblanc** [comment]
I personally disapprove creating news releases until the feature has been tested from our QA team. It's required at least the 'quality check ok' label to go further. Process are keys points espacially from new features.

**Comment by neolitec** [comment]
I also disapprove but this not easily testable in staging unless you put the good breakpoint and you call the good method to enable the feature (disabled by default in staging). I would like to let the QA team to do that but I don't think it's pretty easy for them.
This feature will be testable in  a pre-preproduction environment only so that we must merge before.

**Comment by JDownloader** [comment]
I don't think the role of QA team is to test things like tracking, if a dev have concern about it, he/she should pull the code on his laptop and do some testing. In our setup tracking in only ON for prod and preprod, risk is very low since tracking makes no visual difference to the end client.

**Comment by antleblanc** [comment]
ping @neolitec @whispyy, still has a conflict.

**Comment by neolitec** [comment]
Done

**Comment by neolitec** [comment]
%*&%@
Sorry for that.

File: package.json [hunk 1]
```diff
@@ -68,7 +68,7 @@
     "ng-ckeditor": "esvit/ng-ckeditor#^0.2.1",
     "ng-slide-down": "TheRusskiy/ng-slide-down#^1.0.0",
     "office-ui-fabric-react": "^4.35.2",
-    "ovh-angular-actions-menu": "^3.0.0",
```
Comment (antleblanc) [inline package.json]: `yarn.lock` must be updated as well!

**Comment by jleveugle** [comment]
Hello @neolitec,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

File: package.json (hunk 1 above)
Comment (whispyy) [reply package.json]: Hello @antleblanc as I can see it seems correct in yarn.lock.
Can you check it pls ?

Comment (antleblanc) [reply package.json]: Actually you are trying to bump `ovh-angular-actions-menu` from `v3.0.0` to `v3.2.0`. Must be reflected in diff on `yarn.lock` file isn't it?

Checking on `develop` branch and it's locked to `v3.1.1` (https://github.com/ovh-ux/ovh-manager-dedicated/blob/develop/yarn.lock#L4924).

Hope it's help you!

Comment (antleblanc) [reply package.json]: @whispyy no problem! glad to help!

**Pull request merged**